cmd/soroban-cli/src/commands/contract/deploy.rs

```diff
@@ -220,31 +222,10 @@ fn build_create_contract_tx(
         cond: Preconditions::None,
         memo: Memo::None,
         operations: vec![op].try_into()?,
-        ext: TransactionExt::V1(SorobanTransactionData {
-            resources: SorobanResources {
-                footprint: LedgerFootprint {
-                    read_only: vec![ContractCode(LedgerKeyContractCode { hash })].try_into()?,
-                    read_write: vec![ContractData(LedgerKeyContractData {
-                        contract_id: Hash(contract_id.into()),
-                        key: ScVal::LedgerKeyContractExecutable,
-                    })]
-                    .try_into()?,
-                },
-                // TODO: what values should be used here?
-                instructions: 0,
-                read_bytes: 0,
-                write_bytes: 0,
-                extended_meta_data_size_bytes: 0,
-            },
-            // TODO: what value to use here?
-            refundable_fee: 0,
-            ext: ExtensionPoint::V0,
-        }),
+        ext: TransactionExt::V0,
```

Comment:
woot? I am surprised this works at all

Reply:
We replace this with the real one returned from simulateTransaction before we submit the transaction. So this v0 bit doesn't actually matter.

Reply:
I had debated adding a comment about that, but it felt like 3 different places to add it, so...